routes/sample.js

```diff
@@ -0,0 +1,35 @@
+// FIXME: delete sample router file
+
+const express = require('express');
+
+const sampleRouter = express.Router();
+
+const { keysToCamel } = require('../common/utils');
+
+sampleRouter.use(express.json());
+
+sampleRouter.get('/', async (req, res) => {
+  try {
+    // Query database
+    const data = { };
+
+    res.status(200).json(keysToCamel(data));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+sampleRouter.post('/', async (req, res) => {
+  try {
+    const { _ } = req.body;
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'_' is assigned a value but never used.